app/scripts/modules/core/src/presentation/main.less

```diff
@@ -293,27 +279,33 @@ html {
   }
 
   .detail-content {
-    overflow: visible;
-    margin: 0;
-    @media (max-width: 768px) {
-      top: 170px;
-      width: 95%;
+    z-index: var(--layer-high);
+    position: fixed;
```

Comment:
have you tested if this works with projects page and/or when a blesk is active?

Reply:
No I have not, and will definitely do that.